src/yea_wandb/artifact_emu.py

```diff
@@ -60,6 +61,30 @@ def create(self, variables):
 
         return response
 
+    def link(self, variables):
+        pfolio_name = variables.get("artifactPortfolioName")
+        artifact_id = variables.get("artifactID")
+        if artifact_id is None:
+            artifact_id = variables.get("clientID")
```

Comment:
can we assume that at lease one won't be `None`?

Reply:
Yep, I'll throw a ValueError if it doesn't exist (along with a check for portfolioName